integration/run_test.go

```diff
@@ -287,6 +287,12 @@ func setUp(t *testing.T) (kubernetes.Interface, *v1.Namespace, func(t *testing.T
 	if out, err := integration_util.RunCmdOut(cmd); err != nil {
 		t.Fatalf("testing error: %v\nout: %s", err, out)
 	}
+
+	cmd = exec.Command("kubectl", "apply", "-f", "kritis-server/kritis-config.yaml")
```

Comment:
I think the path here needs to be `testdata/kritis-server/kritis-config.yaml`

Reply:
Good catch, thanks!